src/content-scope-features.js

```diff
@@ -41,6 +42,10 @@ export function load (args) {
         return
     }
 
+    const featureNames = typeof import.meta.injectName === 'string'
+        ? platformSupport[import.meta.injectName]
+        : []
```

Comment:
here I'm passing through what I call the `injectName` (open to better naming ideas) - we need this because `args.platform.name` is not quite the thing we want here. We want to reference the platform name based on when it was built, not the runtime. 

runtime from the apple build:

- args.platform.name = 'ios`
- args.platform.name = 'macos`